src/Domain/ElementId.hpp

```diff
@@ -24,19 +24,21 @@ template <size_t VolumeDim>
 class ElementId {
  public:
   /// Default constructor needed for Charm++ serialization.
-  ElementId() = default;
+  constexpr ElementId() noexcept = default;
```

Comment:
This doesn't compile - figure out noexcept-ness of make_array as this is used to default construct a member variable of ElementId